cfme/fixtures/cli.py

```diff
@@ -249,40 +251,184 @@ def ha_appliances_with_providers(ha_multiple_preupdate_appliances, app_creds):
-    command_set = ("ap", "", "6", "2", "1", "2", "", "", pwd, pwd, app0_ip, app1_ip, "y",
-                   TimedCommand("y", 90), "")
-    apps1.appliance_console.run_commands(command_set)
-    # Configure automatic failover on EVM appliance
-    command_set = ("ap", "", "8", TimedCommand("1", 30), "")
-    apps2.appliance_console.run_commands(command_set)
+    interaction = SSHClientInteraction(apps0.ssh_client, timeout=10, display=True,
+                                       output_callback=logging_callback(apps0))
+    interaction.send('ap')
+    interaction.expect('Press any key to continue.', timeout=20)
+    interaction.send('')
+    interaction.expect('Choose the advanced setting: ')
+    # Configure Database Replication
+    interaction.send('6' if apps1.version < '5.10' else '8')
+    interaction.expect('Choose the database replication operation: ')
+    interaction.send('1')
+    interaction.expect('Enter the number uniquely identifying '
+                       'this node in the replication cluster: ')
+    interaction.send('1')
+    interaction.expect('Enter the cluster database name: |vmdb_production| ')
+    interaction.send('')
+    interaction.expect('Enter the cluster database username: |root| ')
+    interaction.send('')
+    interaction.expect('Enter the cluster database password: ')
+    interaction.send(pwd)
+    interaction.expect('Enter the cluster database password: ')
+    interaction.send(pwd)
+    interaction.expect('Enter the primary database hostname or IP address: |.*| ')
+    interaction.send(app0_ip)
+    interaction.expect('Apply this Replication Server Configuration\? \(Y\/N\): ')
+    interaction.send('y')
+    interaction.expect('Press any key to continue.')
+    interaction.send('')
+
+    # Configure secondary (standby) replication node
+    interaction = SSHClientInteraction(apps1.ssh_client, timeout=10, display=True,
+                                       output_callback=logging_callback(apps1))
+    interaction.send('ap')
+    interaction.expect('Press any key to continue.', timeout=20)
+    interaction.send('')
+    interaction.expect('Choose the advanced setting: ')
+    # Configure Database Replication
+    interaction.send('6' if apps1.version < '5.10' else '8')
+    interaction.expect('Choose the database replication operation: ')
+    interaction.send('2')  # Configure Server as Standby
+    interaction.expect('Choose the encryption key: |1| ')
+    interaction.send('2')
+    interaction.send(app0_ip)
+    interaction.expect('Enter the appliance SSH login: |root| ')
+    interaction.send('')
+    interaction.expect('Enter the appliance SSH password: ')
+    interaction.send(pwd)
+    interaction.expect('Enter the path of remote encryption key: |/var/www/miq/vmdb/certs/v2_key|')
+    interaction.send('')
+    interaction.expect('Choose the standby database disk: |1| ')
+    interaction.send('1' if apps1.version < '5.10' else '2')
+    # "Enter " ... is on line above.
```

Comment:
@jaryn 
I think, you can remove check for ```< 5.10```. As we are not checking below 5.10 versions against branch ```master```

Reply:
I am not so sure it is wise to do it right now. Especially in this commit. There may be some upgrade (migration) tests that need to boot 5.9 cfme and update and migrate to 5.10.